pkg/http/api.go

```diff
@@ -1880,7 +1870,7 @@ func getOrDefaultCapabilites(dict map[string][]string, key string) []string {
 func (a *api) onPutMetadata(reqCtx *fasthttp.RequestCtx) {
 	key := fmt.Sprintf("%v", reqCtx.UserValue("key"))
 	body := reqCtx.PostBody()
-	a.extendedMetadata.Store(key, string(body))
+	_ = a.extendedMetadata.MetadataSet(key, string(body))
```

Comment:
Please handle error properly instead of swallowing it here? I can see that it's always returning nil for now though... So why do we need it, is it for some future change?